common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIdConnectPromptParameter.java

```diff
@@ -71,12 +71,12 @@ public static OpenIdConnectPromptParameter _fromPromptBehavior(final String prom
             case "Auto":
             case "REFRESH_SESSION":
             case "Always":
-                return OpenIdConnectPromptParameter.SELECT_ACCOUNT;
+                return OpenIdConnectPromptParameter.NONE;
```

Comment:
Can we drop a comment in here to explain what appears to be otherwise counterintuitive behavior?